src/main/java/co/cask/hydrator/plugin/common/MacroParser.java

```diff
@@ -0,0 +1,218 @@
+/*
```

Comment:
This code originally resides in app-fabric. It is better not to include app-fabric code in a plugin.